core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -136,7 +140,7 @@ $(document).ready(function() {
             (attempt.hasOwnProperty("attemptId") ? attempt["attemptId"] + "/" : "") + "logs";
           attempt["durationMillisec"] = attempt["duration"];
           attempt["duration"] = formatDuration(attempt["duration"]);
-          var app_clone = {"id" : id, "name" : name, "num" : num, "attempts" : [attempt]};
+          var app_clone = {"id" : id, "name" : name, "version": version, "num" : num, "attempts" : [attempt]};
```

Comment:
I think we can simply use
```
"version": attempt["appSparkVersion"]
```
here.

Any reason we always set the version as the first attempt?